Cargo.toml

```diff
@@ -12,7 +12,7 @@ description = "Authentication extension and backend for Trussed"
 
 [dependencies]
 serde = { version = "1", default-features = false }
-serde-byte-array = "0.1.0"
+serde-byte-array = "0.1.1"
```

Comment:
Why not `0.1.2`? That way we have the `Copy` impl.

Reply:
Do we need it in this crate?

Reply:
We need it in #10

Reply:
I see.  I’ve bumped it to 0.1.2.